aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -467,15 +540,25 @@ def _not_found(self, method: str) -> ResponseBuilder:
         return ResponseBuilder(
             Response(
                 status_code=404,
-                content_type="application/json",
+                content_type=APPLICATION_JSON,
                 headers=headers,
                 body=json.dumps({"message": "Not found"}),
             )
         )
 
     def _call_route(self, route: Route, args: Dict[str, str]) -> ResponseBuilder:
         """Actually call the matching route with any provided keyword arguments."""
-        return ResponseBuilder(self._to_response(route.func(**args)), route)
+        try:
+            return ResponseBuilder(self._to_response(route.func(**args)), route)
+        except ServiceError as e:
+            return ResponseBuilder(
+                Response(
+                    status_code=e.status_code,
+                    content_type=APPLICATION_JSON,
+                    body=json.dumps({"message": str(e)}),
```

Comment:
Thoughts on including the status code in a “code” property as well?

Reply:
@jplock - in the json response? That could work. I was looking at the standard errors coming back from the api gateway

Reply:
Yea putting the status code also in the json response for clients

Reply:
Cool. @jplock do you think it should be the exception class name (like in Chalice?) or that http code? Or something custom that you can define?

Reply:
I thought the value of “e.status_code” for now